moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h

```diff
@@ -150,12 +150,15 @@ private Q_SLOTS:
   void approximateIKChanged(int state);
 
   // Planning tab
+  void computeCartecianPlanButtonClicked();
```

Comment:
This misspelling of "Cartesian" occurs a few more times in the code and PR title, and a few other ones in the code. I wouldn't bother mentioning this normally, but since it is a function name and inconsistent with the function it calls, I think this is a bit more important than a regular typo.

Regardless, I recommend VS Code's Code Spell Checker extension, which I also use :)

Reply:
Thanks, fixed typos.